day-01/src/app.js

```diff
@@ -1,39 +1,34 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import Button from './button';
+import Display from './display';
 
 class Stateful extends React.Component {
   constructor() {
```

Comment:
in the new versions u don't need constructor to play with states just simply use:
```js
state = {
 count: 0,
};
```